README.md

```diff
@@ -31,6 +31,11 @@ make all
 make TARGET=Linux apps
 ```
 
+To compile without using Docker, use the `DISABLE_DOCKER=y` option : 
```

Comment:
Instead of documenting it here, please document this in `Make options` table below in the readme to keep all makefile options together.

Reply:
Done.